docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,64 @@
+= Parameters
+
+The parent key for all of the following parameters is `metallb`.
+
+
+== `namespace`
+
+[horizontal]
+type:: string
+default:: `syn-metallb`
+
+
+
+== `speaker.secretkey`
+
+[horizontal]
+type:: string
+default:: Vault reference
+
+A binary secret key encoded base64.
+It can be generated using the command `openssl rand -base64 128`.
+
+== `addresses`
+
+[horizontal]
+type:: list
+default:: null
```

Comment:
This is strictly speaking not true, as the absence of the key and the explicit value `null` don't have the same behavior for the component implementation.

Reply:
Removed the default as there is no.